components/zcash_address/src/kind/unified.rs

```diff
@@ -116,6 +126,10 @@ impl fmt::Display for ParseError {
             ParseError::InvalidTypecodeValue(v) => write!(f, "Typecode value out of range {}", v),
             ParseError::InvalidEncoding(msg) => write!(f, "Invalid encoding: {}", msg),
             ParseError::OnlyTransparent => write!(f, "UA only contains transparent items"),
+            ParseError::NotUnified => write!(f, "Address is not Bech32m encoded"),
+            ParseError::UnknownPrefix(s) => {
+                write!(f, "Unrecognized Bech32m human-readable prefix: {}", s)
```

Comment:
What error is produced for a string that is valid Bech32 but has an unrecognized prefix?

Reply:
That will be `encoding::ParseError::NotZcash`